src/configuration.js

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+import SpanContext from './span_context';
+import Span from './span';
+import ConstSampler from './samplers/const_sampler';
+import InMemoryReporter from './reporters/in_memory_reporter';
+import ProbabilisticSampler from './samplers/probabilistic_sampler';
+import RateLimitingSampler from './samplers/ratelimiting_sampler';
+import RemoteReporter from './reporters/remote_reporter';
+import CompositeReporter from './reporters/composite_reporter';
+import LoggingReporter from './reporters/logging_reporter';
+import RemoteSampler from './samplers/remote_sampler';
+import Tracer from './tracer';
+import UDPSender from './reporters/udp_sender';
+import {Validator} from 'jsonschema';
+import Logtron from 'logtron';
```

Comment:
This is a default logger, for our uses.  So the user can still pass in a logger that supports "info", "error", "debug", "warn".  I'll move it to prod dependencies though.